src/components/Search/SearchContainer.module.scss

```diff
@@ -109,3 +109,20 @@
 .suggestion:hover .remove {
   display: block;
 }
+
+.header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  background: var(--athens);
+  border: 1px solid var(--procelain);
+  box-sizing: border-box;
+  padding: 12px 16px;
+  color: var(--waterloo);
```

Comment:
Can be achieved using [`className='c-waterloo'`](https://github.com/santiment/san-webkit/blob/master/stories/Introduction.stories.mdx)